src/Serialization/Fields.jl

```diff
@@ -0,0 +1,50 @@
+################################################################################
+# non-fmpz variant
+function save_intern(s::SerializerState, F::Nemo.GaloisField)
+    return Dict(
+        :characteristic => UInt64(characteristic(F))
+    )
+end
+
+function load_intern(s::DeserializerState, F::Type{Nemo.GaloisField}, dict::Dict)
+    return F(UInt64(dict[:characteristic]))
```

Comment:
Note that `F === Nemo.GaloisField` here, so you could rewrite this code as saying
```suggestion
function load_intern(s::DeserializerState, ::Type{Nemo.GaloisField}, dict::Dict)
    return Nemo.GaloisField(UInt64(dict[:characteristic]))
```
Similar elsewhere.